client/src/constants/constants.js

```diff
@@ -0,0 +1,2 @@
+export const PROFILE = "profile";
```

Comment:
I suggest changing these to be something like this

```
export const LOCAL_STORAGE_KEYS = {
     PROFILE: "profile"
}
```

alternatively, you can store these under a file called `localStorageKeys.js` so that it's split off from the other constants, but I'll leave that to you